src/openforms/forms/tests/test_serializers.py

```diff
@@ -83,6 +89,33 @@ def test_date_variable_with_non_trivial_json_trigger(self):
 
         self.assertTrue(serializer.is_valid())
 
+    def test_registration_backend_action_errors_no_value(self):
+        form = FormFactory.create()
+        context = {"request": None, "form_variables": FormVariableWrapper(form=form)}
+
+        serializer = LogicComponentActionSerializer(
+            data={
+                "json_logic_trigger": {
+                    "==": [
+                        {"var": "someDate"},
+                        {},
+                    ]
+                },
+                "action": {
+                    "type": "set-registation-backend",
+                },
+            },
+            context=context,
+        )
+
+        with self.assertRaises(ValidationError) as raised:
+            serializer.is_valid(raise_exception=True)
+
+        self.assertEqual(
+            raised.exception.args[0]["action"]["type"][0].code,
+            "invalid_choice",
+        )
```

Comment:
why not:

```py
is_valid = serializer.is_valid()

self.assertFalse(is_valid)
# assert on serializer.errors["action"]["type"]
```
the exception flow is DRF internals (as the conversion of the exception to an error response is handled entirely in DRF itself even and *could* be subject to change).